program.c

```diff
@@ -11,6 +11,10 @@
 #ifdef JULIA_DEFINE_FAST_TLS // only available in Julia 0.7+
 JULIA_DEFINE_FAST_TLS()
 #endif
+// TODO(nhdaly): Would a better name be JL_IMAGE_NAME?
+#ifndef JULIA_LIB_NAME // Should be specified by -DJULIA_LIB_NAME="libname"
+#define JULIA_LIB_NAME "libhello"
+#endif
```

Comment:
2. Is this unnecessarily safety-conscious? It might be better simply to crash with "JL_IMAGE_NAME not defined" if it's not specified, rather than defaulting to "libhello", which could ultimately be more confusing than helpful. The reason I did this was to ensure we don't break anyone's current behavior, in case they're doing anything custom/weird with `program.c`, but honestly I think it'd be better simply to assume it's defined by `juliac.jl`? (maybe with a comment explaining that it's expected.) what do you think?

Reply:
Done.